pygeoapi/flask_app.py

```diff
@@ -110,14 +118,15 @@ def schemas(path):
                                    mimetype=get_mimetype(basename_))
 
 
+# TODO: inline in execute_from_flask when all views have been refactored
```

Comment:
Is the completion of this TODO not in scope of this PR? I would think it is, but I'm not sure - I was going to ask about the reason for keeping and using `get_response()` in some places while using `execute_from_flask()` in others, making a similar point about having two similar but different functions as I did for the starlette app.

Reply:
@totycro ^^

Reply:
I think this is really the same discussion as https://github.com/geopython/pygeoapi/pull/1405#discussion_r1545755291